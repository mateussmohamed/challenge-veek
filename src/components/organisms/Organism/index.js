import styled from 'styled-components'

const Organism = styled.div`
  background-color: yellowyarn;
`

export default Organism
