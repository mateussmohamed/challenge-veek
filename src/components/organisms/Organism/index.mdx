---
name: Organism
menu: Organisms
---

import { Playground, PropsTable } from 'docz'
import Organism from './index'

# Organism
<PropsTable of={Organism} />

<Playground>
    <Organism>Default</Organism>
</Playground>
