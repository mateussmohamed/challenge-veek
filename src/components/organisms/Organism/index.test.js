import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import Organism from './index'

describe('[<Organism />]', () => {
  const wrap = (props = {}) => shallow(<Organism {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a Organism' })

    expect(wrapper).toMatchSnapshot()
  })
})
