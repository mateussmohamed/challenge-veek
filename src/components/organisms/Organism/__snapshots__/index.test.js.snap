// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Organism />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "Organism-wxahqb-0",
        "isStatic": true,
        "rules": Array [
          "background-color:yellow;",
        ],
      },
      "displayName": "Organism",
      "render": [Function],
      "styledComponentId": "Organism-wxahqb-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a Organism
</StyledComponent>
`;
